lib/pf/Portal/ProfileFactory.pm

```diff
@@ -33,6 +35,8 @@ Create a new pf::Portal::Profile instance based on parameters given.
 
 =cut
 
+tie our $PROFILE_FILTER_ENGINE , 'pfconfig::cached_scalar' => 'resource::ProfileFilterEngine';
```

Comment:
Julien I am assuming this should also use the new standard FilterEngine::Profile